components/inputs/README.md

```diff
@@ -0,0 +1,40 @@
+# Inputs
+
+There are various input components available:
+
+- - [Text](#text-inputs)
+
+## Text Inputs
+
+The `<d2l-input-text>` element is a simple wrapper around the native `<input type="text">` tag. It's intended primarily for inputting generic text, email addresses and URLs.
+
+![example screenshot of text input](./screenshots/text.png?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-text.js';
+</script>
+<d2l-input-text value="hello"></d2l-input-text>
+```
+
+**Properties:**
+
+- `aria-invalid` (optional, String): indicates that the input value is invalid
+- `aria-label` (optional, String): sets an accessible label
+- `autocomplete` (optional, String): specifies which types of values [can be autofilled](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) by the browser
+- `autofocus` (optional, Boolean): when set, will automatically place focus on the input
```

Comment:
Might just add bulk, but should we specify the default values for booleans?

Reply:
I like that -- will add.

Reply:
Actually... so for booleans it should be super clear since the absence of an attribute means it's `false`.

Reply:
Right, so I guess we should always default to false, and word the attribute that way (example: `disabled` if it's enabled by default, and `enabled` if disabled by default)

Reply:
Yeah exactly.